app/bundles/LeadBundle/Assets/js/lead.js

```diff
@@ -535,6 +538,10 @@ Mautic.leadfieldOnLoad = function (container) {
 
 Mautic.updateLeadFieldProperties = function(selectedVal) {
     var defaultValueField = mQuery('input#leadfield_defaultValue');
+
+    if (selectedVal == 'multiselect') {
```

Comment:
Combine this with line 545 `if (selectedVal == 'lookup' || selectedVal == 'multiselect') {`